### ISSUE TITLE
v0.0.116 ChatVectorDBChain with ChatOpenAI fails with `Should always be something for OpenAI.`

### ISSUE THREAD

**Issue by danielchalef** [hwchase17/langchain#1799]
`ChatVectorDBChain` with `ChatOpenAI` fails with `Should always be something for OpenAI.`

It appears `_combine_llm_outputs` added to `ChatOpenAI` in v0.0.116 is receiving a `llm_outputs` list containing a `None` value.

**Comment by LaxmanSinghTomar** [comment]
Yes, facing the same issue with QA with Sources Chain as well for both ChatOpenAI and AzureChatOpenAI.

**Comment by hwchase17** [comment]
ah sorry - will fix
